src/wrappers/themis/wasm/src/secure_cell_seal.ts

```diff
@@ -74,7 +74,7 @@ export class SecureCellSeal {
     return new SecureCellSealWithPassphrase(passphrase);
   }
 
-  encrypt(message: Uint8Array) {
+  encrypt(message: Uint8Array, context: Uint8Array = new Uint8Array()) {
```

Comment:
wouldn't it be a breaking change for users that use `0.14.0`?

maybe we should **add** a new method `encrypt(message, context)` while leaving old method `encrypt(message)`?

Reply:
we have tested, no compatibility issues here.